### DISCUSSION TITLE
Add option to passthrough GPU/iGPU for hardware acceleration in Jellyfin/Plex

### DISCUSSION THREAD

**Discussion by c0bnut** [tteck/Proxmox#344] [Ideas]
Hardware acceleration improves transcoding performance and takes load off the CPU. Also is there a reason Ubuntu is used as the template instead of Debian?

https://jellyfin.org/docs/general/administration/hardware-acceleration.html
https://ashu.io/blog/media-server-lxc-proxmox/#intel

**Comment by tteck** [comment]
> Plex Media Server LXC
With Hardware Acceleration Support

Jellyfin has the same

You can read through the Change log and find answers
Also, https://github.com/tteck/Proxmox/issues/8 started it all 😄
